presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTaskFactory.java

```diff
@@ -101,6 +104,9 @@ public class HttpRemoteTaskFactory
     private final MetadataManager metadataManager;
     private final QueryManager queryManager;
     private final DecayCounter taskUpdateRequestSize;
+    // TODO: use config file to set this value
+    private final EventLoopGroup eventLoopGroup = new DefaultEventLoopGroup(2000,
```

Comment:
Probably not worth setting it to a value higher then the number of CPU's available on the system

Reply:
Good point!